pypeman/remoteadmin.py

```diff
@@ -584,11 +583,11 @@ async def start(self):
         # redirect to index.html
         app.router.add_get('/', self.redirect_to_index)
 
-        app.router.add_static(
-            '/',
-            path=os.path.join(client_dir),
-            name='static'
-        )
+        # app.router.add_static(
+        #     '/',
+        #     path=os.path.join(client_dir),
+        #     name='static'
+        # )
```

Comment:
it seems remoteadmin interface and remote admin plugin fight for the same url 'space'
do we ever need both at the same time?